dev-requirements.txt

```diff
@@ -3,85 +3,162 @@
 -e git+https://github.com/nucypher/py-solc.git@391b8da1a6bac5816877197bda25527c6b0b8c15#egg=py-solc
```

Comment:
This file seems to have an unusually large amount of new requirements... what's up with that?

Reply:
Are these duplicates of what's in requirements.txt?

Reply:
Hmmm - checking. :thinking:

Reply:
Upon review, I think they look correct, although I am also wondering why there are so many new lines.